src/client/clientlauncher.cpp

```diff
@@ -357,6 +376,7 @@ bool ClientLauncher::launch_game(std::string &error_message,
 	// Initialize menu data
 	MainMenuData menudata;
 	menudata.address                         = address;
+	menudata.serverdescription               = server_description;
```

Comment:
please pass in server name rather than server state

Reply:
same